### PR TITLE
Bump avago anr

### PR DESCRIPTION
**Pull request by felipemadero**

### CODE REVIEW

File: client/p.go [hunk 1]
```diff
@@ -203,55 +204,22 @@ func (pc *p) GetValidator(ctx context.Context, rsubnetID ids.ID, nodeID ids.Shor
...
-			return time.Time{}, time.Time{}, fmt.Errorf("%w: %T %+v", ErrInvalidValidatorData, v, v)
-		}
-		nodeIDs, ok := va["nodeID"].(string)
-		if !ok {
-			return time.Time{}, time.Time{}, ErrInvalidValidatorData
-		}
-		if nodeIDs == nodeID.PrefixedString(constants.NodeIDPrefix) {
-			validator = va
-			break
+		if v.NodeID == nodeID {
```
Comment (patrick-ogrady) [inline client/p.go:208]: much cleaner lol

File: scripts/tests.e2e.sh [hunk 2]
```diff
@@ -20,64 +20,70 @@ fi
...
 
 #################################
 echo "building e2e.test"
 # to install the ginkgo binary (required for test build and run)
 go install -v github.com/onsi/ginkgo/v2/ginkgo@v2.0.0
+# Set the CGO flags to use the portable version of BLST                                                                 
+#                                                                                                                       
+# We use "export" here instead of just setting a bash variable because we need                                          
+# to pass this flag to all child processes spawned by the shell.                                                        
+export CGO_CFLAGS="-O -D__BLST_PORTABLE__"
```
Comment (patrick-ogrady) [inline scripts/tests.e2e.sh:69]: Could we also add this to `./scripts/build.sh`

File: scripts/tests.unit.sh [hunk 3]
```diff
@@ -6,4 +6,9 @@ if ! [[ "$0" =~ scripts/tests.unit.sh ]]; then
   exit 255
 fi
 
+# Set the CGO flags to use the portable version of BLST                                                                 
+#                                                                                                                       
+# We use "export" here instead of just setting a bash variable because we need                                          
+# to pass this flag to all child processes spawned by the shell.                                                        
+export CGO_CFLAGS="-O -D__BLST_PORTABLE__"
```
Comment (patrick-ogrady) [inline scripts/tests.unit.sh:13]: can you leave a new line between this export and the next line?

File: scripts/tests.e2e.sh (hunk 2 above)
Comment (patrick-ogrady) [reply scripts/tests.e2e.sh:69]: should also put a new line before and after this chunk

Comment (felipemadero) [reply scripts/tests.e2e.sh:69]: there is not `build.sh` script but can create a little one

File: .github/workflows/build-test-release.yml [hunk 4]
```diff
@@ -19,22 +19,43 @@ jobs:
...
+        with:
+          repository: tpoechtrager/osxcross
+          path: osxcross
+      - name: Build osxcross
+        run: |
+          sudo apt-get -y install clang llvm-dev libxml2-dev uuid-dev libssl-dev bash patch make tar xz-utils bzip2 gzip sed cpio libbz2-dev
+          cd osxcross
+          wget https://github.com/phracker/MacOSX-SDKs/releases/download/11.3/MacOSX11.3.sdk.tar.xz -O tarballs/MacOSX11.3.sdk.tar.xz
+          echo cd4f08a75577145b8f05245a2975f7c81401d75e9535dcffbb879ee1deefcbf4 tarballs/MacOSX11.3.sdk.tar.xz | sha256sum -c -
+          UNATTENDED=1 ./build.sh
```
Comment (gyuho) [inline .github/workflows/build-test-release.yml:50]: 🙏

File: .github/workflows/build-test-release.yml [hunk 5]
```diff
@@ -19,22 +19,43 @@ jobs:
...
       - name: Run unit tests
         shell: bash
-        run: scripts/tests.unit.sh
+        run: |
+          cd subnet-cli
+          scripts/tests.unit.sh
       - name: Run e2e tests
         shell: bash
-        run: scripts/tests.e2e.sh 1.7.6
+        run: |
```
Comment (gyuho) [inline .github/workflows/build-test-release.yml:34]: Q. If we 1.9 both for e2e and go module, would it break?

File: tests/e2e/e2e_test.go [hunk 6]
```diff
@@ -83,9 +82,6 @@ var _ = ginkgo.BeforeSuite(func() {
 	cancel()
 	gomega.Ω(err).Should(gomega.BeNil())
 
-	// start is async, so wait some time for cluster health
-	color.Outf("{{green}}waiting for healthy{{/}}\n")
-	time.Sleep(time.Minute)
```
Comment (gyuho) [inline tests/e2e/e2e_test.go:88]: nice

**Review by gyuho — APPROVED** [review]

File: .github/workflows/build-test-release.yml (hunk 5 above)
Comment (felipemadero) [reply .github/workflows/build-test-release.yml:34]: did not tried, just used the avago bumped by anr bump. will check. e2e should work.

Comment (felipemadero) [reply .github/workflows/build-test-release.yml:34]: worked. bumping

**Review by patrick-ogrady — APPROVED** [review]

**Pull request merged**